packages/frontity/src/cli/index.ts

```diff
@@ -26,26 +26,19 @@ tsNode.register({
 });
 
 import program from "commander";
-import { readFileSync } from "fs-extra";
-import { resolve } from "path";
-import {
-  create,
-  dev,
-  build,
-  serve,
-  subscribe,
-  info,
-  unknown,
-  createPackage
-} from "./actions";
+import { dev, build, serve } from "../commands";
 
-const { version } = JSON.parse(
-  readFileSync(resolve(__dirname, "../package.json"), { encoding: "utf8" })
-);
+import { default as create } from "./create";
+import { default as createPackage } from "./create-package";
+import { default as subscribe } from "./subscribe";
+import { default as unknown } from "./unknown";
+import { default as info } from "./info";
+
+import packageJson from "../../package.json";
```

Comment:
We can just import the JSON file, the only drawback is that it creates `src` inside the `dist`folder.